google-cloud-spanner/src/test/java/com/google/cloud/spanner/OptionsTest.java

```diff
@@ -510,4 +517,68 @@ public void testTransactionOptionsPriorityEquality() {
     assertNotEquals(options2, options3);
     assertNotEquals(options2, options4);
   }
+
+  @Test
+  public void updateOptTest() {
```

Comment:
nit: please don't abbreviate `Options` to `opts`. (I know this was already like this, but if you are to make any other changes, could you please fix this as well).

Reply:
Done.